src/data_source/storage/sql.rs

```diff
@@ -326,6 +340,8 @@ pub struct SqlStorage {
     client: Arc<Client>,
     tx_in_progress: bool,
     kill: Option<oneshot::Sender<()>>,
+    pruner_cfg: Option<PrunerCfg>,
+    pruned_height: Option<u64>,
```

Comment:
Maybe we can get rid of this since we are storing the same thing at a higher level in `NotifyStorage`